volume_project/extract/2022/rescu_sqls/03_vol_threshold_check.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE scannon.rescu_enuf_vol_22 AS (
```

Comment:
Considered changing the threshold to 6000 per lane per day... discussed with @radumas and he said we should keep the thresholds where they are since they are already low.